opencti-platform/opencti-front/src/private/components/profile/Notifications.tsx

```diff
@@ -280,21 +255,33 @@ const Notifications: FunctionComponent = () => {
       label: 'Message',
       percentWidth: 48,
       isSortable: false,
-      render: ({ notification_content }: NotificationsLine_node$data) => {
-        const events = notification_content.map((n) => n.events).flat();
+      render: (data: NotificationsLine_node$data) => {
+        const events = data.notification_content.map((n) => n.events).flat();
         const firstEvent = events.at(0);
+        const multipleEvents = events.length > 1;
 
-        return (<div style={{ height: 20, fontSize: 13, float: 'left', whiteSpace: 'nowrap', overflow: 'hidden', textOverflow: 'ellipsis', paddingRight: 10 }}>
-          {events.length > 1 ? (
-            <i>{t_i18n('Digest with multiple notifiers')}</i>
-          ) : (
-            <Tooltip title={firstEvent?.message ?? '-'}>
-              <span>
-                <MarkdownDisplay content={firstEvent?.message ?? '-'} remarkGfmPlugin commonmark removeLinks/>
-              </span>
-            </Tooltip>
-          )}
-        </div>);
+        return (
+          <div style={{ height: 20, fontSize: 13, float: 'left', whiteSpace: 'nowrap', overflow: 'hidden', textOverflow: 'ellipsis', paddingRight: 10 }}>
+            {multipleEvents ? (
+              <div
+                onClick={(e) => {
+                  e.preventDefault();
+                  e.stopPropagation();
```

Comment:
You can use `stopEvent` 
https://github.com/OpenCTI-Platform/opencti/blob/404e57eaa09bb09717845888361efe4c55bb7d66/opencti-platform/opencti-front/src/utils/domEvent.ts#L3-L6

```suggestion
                  stopEvent(e);
```